srcpkgs/badwolf/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'badwolf'
 pkgname=badwolf
 version=1.2.2
-revision=1
+revision=2
 build_style=configure
 make_check_target=test
 hostmakedepends="pkg-config gettext"
-makedepends="webkit2gtk-devel libsoup-devel"
+makedepends="libwebkit2gtk41-devel libsoup3-devel"
```

Comment:
> notice: Packagers should consider setting the URI parsing library (guri or libsoup2) in WITH_URI_PARSER

?

Reply:
oh. good catch. I might be blind.
looking back at the logs and configure script, this means that the build with this PR was silently building with the `GUri` backend instead of `libsoup`. there's a few options to consider moving forward now:
1) use `libwebkit2gtk41` with `libsoup2`, something the configure script warns about
2) use `libwebkit2gtk41` with `guri` (the current pr, just would need some template cleaning), which goes against the favoring of the software
3) abandon both routes and keep `webkit2gtk` with `libsoup2`, which the GNOME team seems to be moving away from, though not really any time too soon

I guess it's a no-win situation, but (3) is probably the most preferable for stability right now until GNOME fully moves away from libsoup2.

Reply:
I was led to believe that due to [this commit](https://hacktivis.me/git/badwolf/commit/0199df2a85269e4f7f759a75e7ad89fa7386a1a4.html), but perhaps that truly only applies to `webkit2gtk-4.0`? I guess if it doesn't memleak then it doesn't matter since the software does support both.
I'll try using `webkit2gtk-4.1` and explicitly using `GUri`. Seems to be the move.